tools/driver/swift_symbolgraph_extract_main.cpp

```diff
@@ -260,16 +260,16 @@ int swift_symbolgraph_extract_main(ArrayRef<const char *> Args, const char *Argv
   // don't need to print these errors.
   CI.removeDiagnosticConsumer(&DiagPrinter);
   
-  for (const auto &ModuleName : VisibleModuleNames) {
-    if (ModuleName.str().startswith("_")) {
-      auto CIM = CI.getASTContext().getModuleByName(ModuleName.str());
-      if (CIM && CIM->isCrossImportOverlayOf(M)) {
-        const auto &CIMainFile = CIM->getMainFile(FileUnitKind::SerializedAST);
-        llvm::errs() << "Emitting symbol graph for cross-import overlay module file: "
-          << CIMainFile.getModuleDefiningPath() << '\n';
-        
-        Success |= symbolgraphgen::emitSymbolGraphForModule(CIM, Options);
-      }
+  SmallVector<ModuleDecl *, 2> Overlays;
```

Comment:
Any specific reason for `2`? Should we just omit the second template argument?

Reply:
No specific reason, other than "i thought the template parameter was required". It looks like i can leave the template parameter out entirely and get a ["reasonable default"](https://llvm.org/doxygen/classllvm_1_1SmallVector.html), so i'll do that.

Reply:
TIL and I wish I’d known this years ago